addons/mail/static/src/components/message/message.js

```diff
@@ -448,14 +449,20 @@ class Message extends Component {
             }
             return;
         }
-        this.state.isClicked = !this.state.isClicked;
+        if (!isEventHandled(ev, 'Message.ClickAuthorAvatarHandled')
+            && !isEventHandled(ev, 'Message.ClickAuthorNameHandled')
+            && !isEventHandled(ev, 'Message.ClickFailureHandled')
+           ) {
+            this.state.isClicked = !this.state.isClicked;
+        }
     }
 
     /**
      * @private
      * @param {MouseEvent} ev
      */
     _onClickAuthorAvatar(ev) {
+        markEventHandled(ev, 'Message.ClickAuthorAvatarHandled');
```

Comment:
Don't need to add "Handled" in the name, it makes things a bit too verbose though. We know this helper is for stuff that is handled, we just need to give it contextual name.